tensordict/_lazy.py

```diff
@@ -1552,7 +1548,7 @@ def _multithread_rebuild(
         # We know batch_size is None, this has been checked earlier
         batch_size: Sequence[int] | None = None,
         device: torch.device | None = NO_DEFAULT,
-        names: Sequence[str] | None = None,
+        names: Sequence[str] | None = NO_DEFAULT,
```

Comment:
Since the default value is not `None` anymore, maybe we can drop the `| None` bit ?